### PR TITLE
Feature: Extend rules to allow multiple words in class

### PR DESCRIPTION
**Pull request by thomasparsons**
# Pull request checklist

- [x] The PR has a proper title. Use [Semantic Commit Messages](https://seesparkbox.com/foundry/semantic_commit_messages). (No more branch-name title please)
- [x] Make sure you are requesting the right branch.
- [x] Make sure this is ready to be merged into the relevant branch. Please don't create a PR and let it hang for a few days.
- [x] Ensure all tests are passing.

# PR type

- new feature

# Description

- Support multi word classnames such as `className="oneWord twoWord threeWord"`
- ensure everything still works

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Pull request checklist
```
Comment (lwkchan) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: 👍🏻

**Review by blown-capacitor — DISMISSED** [review]
Can we please have a test where the `__element` and or the `--modifier` are checked for casing too. If this checks that already

**Comment by thomasparsons** [comment]
> Can we please have a test where the `__element` and or the `--modifier` are checked for casing too. If this checks that already

We don't currently have any BEM support in this rule, I think adding it is a good idea, but we need to scope/spec those rules out a little first

**Comment by thomasparsons** [comment]
> > Can we please have a test where the `__element` and or the `--modifier` are checked for casing too. If this checks that already
> 
> We don't currently have any BEM support in this rule, I think adding it is a good idea, but we need to scope/spec those rules out a little first

https://app.clubhouse.io/fundingoptions/story/19878/eslint-rule-support-bem

**Review by blown-capacitor — APPROVED** [review]

**Pull request merged**